### PR TITLE
TRO-762 refactor to homogenize alerts and dashboard copy

### PR DESCRIPTION
**Pull request by fraucamidokura**
Make same changes that we made to use objects in alerts for dashboard

### CODE REVIEW

File: grafana_sync.py [hunk 1]
```diff
@@ -79,6 +79,9 @@ def set_dashboard(base_url, session, data_json, overwrite=False, adapt_uid=False
         raise ValueError("JSON contains no dashboard")
 
     dashboard_json = data_json
+    #Ensure is a createion
+    data_json['dashboard'].pop('id',None)
```
Comment (fraucamidokura) [inline grafana_sync.py:83]: It was not working without this changes

File: grafana_sync.py [hunk 2]
```diff
@@ -79,6 +79,9 @@ def set_dashboard(base_url, session, data_json, overwrite=False, adapt_uid=False
         raise ValueError("JSON contains no dashboard")
 
     dashboard_json = data_json
+    #Ensure is a createion
```
[the remark this answers is not in the document]
Comment (fraucamidokura) [reply grafana_sync.py]: done

File: grafana_sync.py (hunk 1 above)
Comment (fraucamidokura) [reply grafana_sync.py]: Good catch, I did make the change. 

This part of the code is a little weak. On alert, I did find an issue when copying from an environment with manually copied alerts and trying to use the tool. Then it found duplicate alerts as the same title is used but a different id.

I did not work on that, and here more or less we will have the same problem. At the moment I did not put effort here. We need to see how the tool will be used, and then make it grow in that direction.

Comment (LIII-XXII) [reply grafana_sync.py]: OK

**Review by LIII-XXII — APPROVED** [review]

**Pull request merged**